test/comedy_test.exs

```diff
@@ -0,0 +1,11 @@
+defmodule AshPostgres.Test.ComedyTest do
```

Comment:
can we put this into the calculations test module?

Reply:
Done.